src/collaboration.tex

```diff
@@ -30,7 +30,7 @@ \subsection{Academic Outreach and Engagement}
 \clearpage
```

Comment:
This `\clearpage` was put in originally for page continuity, but now needs to get edited.